test/e2e_node/services/kubelet.go

```diff
@@ -249,8 +249,6 @@ func (e *E2EServices) startKubelet(featureGates map[string]bool) (*server, error
 		unitName = fmt.Sprintf("kubelet-%s.service", unitTimestamp)
 		cmdArgs = append(cmdArgs,
 			systemdRun,
-			// Set the environment variable to enable kubelet config drop-in directory.
-			"--setenv", "KUBELET_CONFIG_DROPIN_DIR_ALPHA=yes",
 			"-p", "Delegate=true",
```

Comment:
There is one more place where this variable is mentioned: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L288
Should it be removed from there as well?

Reply:
Fixed